common/ui/keyring/keyRing.js

```diff
@@ -59,8 +59,11 @@ var options = options || null;
 
     options.keyring('getKeys', initKeyringTable);
```

Comment:
Probably getKeys is then also called multiple times. Should fix the root of the problem and prevent that these initializations happen more than once.